common/browser_context.go

```diff
@@ -413,25 +406,19 @@ func (b *BrowserContext) runWaitForEventHandler(
 		case ev := <-chEvHandler:
 			if ev.typ == EventBrowserContextClose {
 				b.logger.Debugf("BrowserContext:WaitForEvent:go():EventBrowserContextClose:return", "bctxid:%v", b.id)
```

Comment:
We might want to update these log messages as well. Like: `"BrowserContext:runWaitForEventHandler():`.

Reply:
Good catch on this. Resolved altogether in af81e9a81d4e02f3501a755fc3edb495f6cf3c3f.

Reply:
Thanks!